static/global.css

```diff
@@ -2,7 +2,32 @@
 
 @font-face {
   font-family: 'Visby CF';
-  src: url('fonts/VisbyCF-Bold.woff2');
+  font-display: block;
+  font-weight: 700;
+  src: url('fonts/VisbyCF-Bold.woff2') format('woff2'),
+    url('fonts/VisbyCF-Bold.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Open Sans';
+  font-weight: 400;
+  font-display: block;
+  src: url('fonts/OpenSans-Regular.ttf');
+}
+
+@font-face {
+  font-family: 'Open Sans';
+  font-weight: 700;
+  font-display: swap;
+  src: url('fonts/OpenSans-Bold.ttf');
+}
+
+@font-face {
+  font-family: 'Open Sans';
+  font-weight: 400;
```

Comment:
`font-weight: normal;`

Reply:
Was trying to be specific about the weight for `@font-face declarations`. Did some research tho and it looks like `normal` directly translates to `400` (and `bold` to `700`). I'll fix all the `global.css` declarations